Clever/who-is-who
# Upgrade node 12 to node 14

**Jira:**
https://app.getcortexapp.com/admin/scorecards/332

**Overview:**
Frontend guild is upgrading node versions across the fleet. This PR is generated by https://github.com/Clever/microplane-scripts/pull/42 

**Testing:**
1. Run `make test` and see error when building with nodeV14
2. Run `nvm use 14` 
3. Make sure the `make test` and `make` pass locally
4. Make sure the tests pass in CI

**Roll Out:**
